fab/fabfile.py

```diff
@@ -849,27 +903,6 @@ def _migrate():
         sudo('%(virtualenv_root)s/bin/python manage.py migrate --noinput' % env)
 
 
-@task
-@roles(ROLES_DB_ONLY)
-def migrate():
```

Comment:
decided this command doesn't make too much sense anymore given we don't have a preindex directory